next-tavla/app/(admin)/edit/[id]/components/TileCard/index.tsx

```diff
@@ -69,6 +69,32 @@ function TileCard({
     const [confirmOpen, setConfirmOpen] = useState(false)
     const [isColumnModalOpen, setIsColumnModalOpen] = useState(false)
 
+    const walkingDistanceInMinutes = Math.ceil(
+        (tile.walkingDistance?.distance ?? 0) / 60,
+    )
+    const [offsetBasedOnWalkingDistance, setOffsetBasedOnWalkingDistance] =
+        useState(walkingDistanceInMinutes === tile.offset)
+
+    const offsetRef = useRef<HTMLInputElement | null>(null)
+
+    useEffect(() => {
+        const updateOffsetInputValue = () => {
+            if (offsetRef.current) {
+                offsetBasedOnWalkingDistance
+                    ? (offsetRef.current.valueAsNumber =
+                          walkingDistanceInMinutes)
+                    : (offsetRef.current.valueAsNumber = tile.offset ?? 0)
+            }
+        }
+        updateOffsetInputValue()
+    }, [offsetBasedOnWalkingDistance, walkingDistanceInMinutes, tile.offset])
+
```

Comment:
Dette kunne vi gjort med en controlled input på offset og en state variabel for å håndtere verdien